sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -2515,7 +2516,7 @@ class Analyzer(override val catalogManager: CatalogManager) extends RuleExecutor
       // Return if a sub-tree does not contain SQLFunctionExpression.
       case p: LogicalPlan if !p.containsPattern(SQL_FUNCTION_EXPRESSION) => p
 
-      case f @ Filter(cond, a: Aggregate)
+      case f@Filter(cond, a: Aggregate)
```

Comment:
the previous code style is more correct

Reply:
Yea let me revert these unrelated changes.